include/clang/AST/ASTContext.h

```diff
@@ -2335,6 +2335,10 @@ class ASTContext : public RefCountedBase<ASTContext> {
   /// differ in some other way than checkedness.
   bool isEqualIgnoringChecked(QualType T1, QualType T2) const;
 
+  /// \brief Return true if this type is a checked type that is not
+  /// allowed to be passed or returned from a no prototype function.
+  bool isNotAllowedForNoProtoTypeFunction(QualType T1) const;
```

Comment:
I'll fix this to be prototype (all lower case).   The clang is inconsistent about the capitalization of prototype.